### DISCUSSION TITLE
能否支持指定一些自定义翻译过滤规则，例如粗体，字体，括号内不进行翻译

### DISCUSSION THREAD

**Discussion by tufeixp** [immersive-translate/immersive-translate#1216] [General]
能否支持指定一些自定义翻译过滤规则，例如粗体，字体，括号内不进行翻译

**Comment by theowenyoung** [comment]
有些情况可能是支持的，但是比较复杂..

https://immersivetranslate.com/docs/advanced/

比如粗体不翻译的话，可以配置 忽略的标签 `"excludeSelectors.add":["STRONG"]`

如果括号内不翻译的话，则可以配置 正则： `"noTranslateRegexp.add":[]`
